linera-chain/src/data_types.rs

```diff
@@ -221,6 +234,7 @@ pub struct BlockProposal {
     pub owner: Owner,
     pub signature: Signature,
     pub hashed_certificate_values: Vec<HashedCertificateValue>,
+    pub blobs: BTreeMap<BlobId, Blob>,
```

Comment:
Not sure if it makes sense to include the IDs in the serialized proposal: The server has to recompute them anyway, so we could save a few bytes. (If it turns out that we need to access the blobs by ID here a lot, we could make it a requirement that they are sorted by (implicit) ID, and memoize the ID as part of the deserialized type.)